packages/pixeloven-docs/src/configs/config.tsx

```diff
@@ -30,6 +31,20 @@ addDecorator(
 );
 addDecorator(withKnobs);
 
+
+/**
+ * Wrapper with custom id for modals to attach to
+ */
+const modalContainerDecorator = (storyFn: () => {}) => (
+    <>
+        <div id= "modal-root" />
```

Comment:
Is the space between `=` and `"` a concern?

Reply:
Should the modal root follow the normal root so it's more like what we'd expect to see in our own project?

Reply:
I have the modal div on top in our project

Reply:
I can remove the space

Reply:
Ah ok, I had misunderstood the demo and thought the modal root was at the end of the `body`.